src/julia/tests/test_juliaoptions.py

```diff
@@ -17,7 +17,7 @@
     (dict(min_optlevel=2), ["--min-optlevel=2"]),
     (dict(threads="auto", optimize=3), ["--optimize=3", '--threads=auto']),
     (dict(optimize=3, threads="auto"), ["--optimize=3", '--threads=auto']),  # passed order doesn't matter
-    (dict(compiled_modules=None, depwarn="yes"), ["--depwarn=yes"]),
+    (dict(compiled_modules="auto", depwarn="yes"), ["--depwarn=yes"]),
```

Comment:
I think this is just the test file, no? I think it needs to be set here: https://github.com/JuliaPy/pyjulia/blob/69b734f117e489904a07d20f46c8f96b56577779/src/julia/options.py#L166-L176